hedera-node/test-clients/src/main/java/com/hedera/services/yahcli/commands/contract/00-TODO.md

```diff
@@ -0,0 +1,13 @@
+
+* selectors
```

Comment:
I don't we should be checking in to-do files.
If there's a top level README you can not the product EPIC ticket there.
In that EPIC issue you can have the list of issues/features and check them off over time